app/src/main/java/com/kickstarter/ApplicationModule.java

```diff
@@ -649,6 +650,8 @@ ExperimentsClientType provideOptimizely(final @ApplicationContext @NonNull Conte
 
     final OptimizelyManager optimizelyManager = OptimizelyManager.builder()
       .withSDKKey(optimizelyKey)
+      .withEventDispatchInterval(60L * 10L)
```

Comment:
What's the default event dispatch interval? 10 minutes seems high, I'd imagine we want to dispatch events fairly frequently?

Reply:
it's `-1` lol
```        // -1 will cause the background download to not be initiated.
        private long datafileDownloadInterval = -1L;
        // -1 will cause the background download to not be initiated.
        private long eventDispatchInterval = -1L;```